project/MicroServiceBuild.scala

```diff
@@ -16,7 +16,7 @@ object MicroServiceBuild extends Build with MicroService {
 private object AppDependencies {
   import play.sbt.PlayImport._
   private val pegdownVersion = "1.4.2"
-  private val scalatestVersion = "2.2.2"
+  private val scalatestVersion = "2.2.6"
```

Comment:
Can we check these versions against TAI-Frontend and adjust that accordingly

Reply:
Checked TAI-Frontend and that is currently using 2.2.6 as well